code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -689,3 +690,16 @@ GLOBAL_LIST_INIT(plastic_recipes, list(
 
 /obj/item/stack/sheet/plastic/five
 	amount = 5
+
+/*
+ * Saltpetre crystal
+ */
+
+/obj/item/stack/sheet/saltpetre_crystal
+	name = "saltpetre crystal"
+	desc = "A bunch of saltpetre crystals"
```

Comment:
```suggestion
	desc = "A bunch of saltpetre crystals."
```
And could maybe think of a more unique description?

Reply:
done